modules/javafx.graphics/src/main/java/com/sun/javafx/font/coretext/CTFontFile.java

```diff
@@ -71,6 +98,12 @@ public synchronized void dispose() {
         }
     }
 
+    @Override
+    public boolean isBold() {
+        // Need to do this until we add font variation support into the super-class
+        return fullName.equals("System Font Bold") || super.isBold();
```

Comment:
same here.  I am sure fullName cannot be null

Reply:
right it should not be possible.